make_profiler/report_export.py

```diff
@@ -0,0 +1,83 @@
+from datetime import datetime
+import json
+DATE_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+
+status_list = {}
+status = []
+
+def export_report(performance, docs):
+    strOutputFile = 'report.json'
+    fo = open(strOutputFile, 'w', encoding="utf-8")
+    n_in_progress = 0
+    n_failed = 0
+    n_total = 0
+    oldest_completed_target = ''
+
+    for key in performance:
+        rec = performance[key]
+        n_total += 1
+        if rec["failed"]:
+            event_type = "failed"
+            n_failed += 1
+        elif rec["done"]:
+            event_type = "completed"
+        elif rec["running"]:
+            event_type = "started"
+            n_in_progress += 1
+        else:
+            event_type = "??unknown??"
+
+        #event_time = datetime.fromtimestamp(int(rec['start_current'])).strftime(DATE_FORMAT)
```

Comment:
which timezone will it be in? you explicitly set Z at the end in format but I don't see where you set that you want date in UTC

Reply:
Bug is fixed. In backend time is formatted as UTC, and on UI is presented according to the client time zone.